.github/workflows/reusable-check-labels-on-issues.yml

```diff
@@ -0,0 +1,91 @@
+on:
+  workflow_call:
+    inputs:
+      label-format-list:
+        description: 'The Regex formats expected for the labels; must be a JSON list'
+        default: '[
+          ".*"
+        ]'
+        type: string
+        required: false
+      label-error-message:
+        description: 'Error message to be posted when the labels set don''t match the required list of formats.'
+        default: 'At least one label is required.'
+        type: string
+        required: false
+      label-success-message:
+        description: 'Message to be posted when the labels set fulfill the entire list of expected formats.'
+        default: '✅ Yay, issue looks great!'
```

Comment:
Do we _need_ a message if everything looks fine?

Reply:
True, we don't need a `default` here.

Reply:
Again going back to this I realise why 😓 

The API via the `gh` CLI is rather limited. It doesn't allow for fetching comments or deleting comments, only adding a comment or editing the latest one. To make the code simpler, avoiding curl and more calls, I use just what's available and that seemed ok for this use case.